### PR TITLE
cephadm: fix typo in postinst instructions

### PR DESCRIPTION
**Pull request by akamac**
fix typo in cephadm user gecos

### CODE REVIEW

**Comment by akamac** [comment]
jenkins test signed

File: debian/cephadm.postinst [hunk 1]
```diff
@@ -25,7 +25,7 @@ case "$1" in
        # 1. create user if not existing
        if ! getent passwd | grep -q "^cephadm:"; then
          echo -n "Adding system user cephadm.."
-         adduser --quiet --system --disabled-password --gecos 'Ceph-dameon user for cephadm' --shell /bin/bash cephadm 2>/dev/null || true
+         adduser --quiet --system --disabled-password --gecos 'Ceph-daemon user for cephadm' --shell /bin/bash cephadm 2>/dev/null || true
```
Comment (mgfritch) [inline debian/cephadm.postinst:28]:
```suggestion
         adduser --quiet --system --disabled-password --gecos 'cephadm user for mgr/cephadm' --shell /bin/bash cephadm 2>/dev/null || true
```



Make this the same on all platforms (rhel, suse, debian)?

This is what's used in the rpm spec:
https://github.com/ceph/ceph/blob/71f75fe41f84a0a255dcb6d607096dea2f83eb1d/ceph.spec.in#L1457

cc @sebastian-philipp

**Comment by sebastian-philipp** [comment]
could you please prefix your commit message title with `cephadm: `. Thanks for your contribution!

**Comment by sebastian-philipp** [comment]
ping @akamac  ?

**Comment by sebastian-philipp** [comment]
see #34820

**Pull request closed without merging**